opentech/apply/projects/forms.py

```diff
@@ -0,0 +1,11 @@
+from django import forms
+
+
+class CreateProjectForm(forms.Form):
+    submission = forms.CharField(widget=forms.HiddenInput())
```

Comment:
Should this be ModelChoice field so that we can skip all the checks in the view?
```
forms.ModelChoiceField(
    queryset=Submissions.objects.filter(project__isnull=True),
    widget=forms.HiddenInput(),
)
```

Reply:
Fixed in 379d6b5d